lib/utils/javascript.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
This file should be under a `helpers` folder since this package calls them helpers not `utils`.